linera-core/src/chain_worker/state/temporary_changes.rs

```diff
@@ -346,20 +349,22 @@ where
         policy: &ResourceControlPolicy,
     ) -> Result<(), WorkerError> {
         ensure!(
```

Comment:
Should this be a `&self`-method of `BlobContent`?

Reply:
Mmm ok `ResourceControlPolicy` is not in `linera-base`

Reply:
Yes. I actually tried to move `Blob` and `BlobContent` to `linera-chain`, but it's used in `linera-execution`. Maybe something to consider for a separate PR.

Also for a separate PR: I was thinking of renaming:
* `BlobContent` → `Blob`
* `Blob` → `HashedBlob`

Reply:
There's a `Hashed<T>` data type that we should use for `HashedBlob` (so maybe a type alias). I'd like to reuse the already-existing setup if possible.

Reply:
Yeah, sounds like `fn lite()` on `Hashed` was a mistake. If you have an idea - be my guest :)

Reply:
I'll just remove it and add `LiteValue::new` instead, that takes a `&Hashed<impl CertificateValueT>`.